### ISSUE TITLE
major version bump of virtualenv (20.0.1, updated 2019-02-10) breaks multistage docker builds using pipenv

### ISSUE THREAD

**Issue by ianlofs** [pypa/virtualenv#1560]
The new version of virtualenv  appears to break python apps deployed using a multistage docker build process, ie.
```
FROM python:3.6 as build
# install any build dependencies using package manager
RUN pip install pipenv
RUN pipenv install
RUN python setup.py install
FROM python:3.6
COPY --from build <path_to_virtualenv_location> <path_to_location_in_final_step>
ENTRYPOINT ['<path_to_location_in_final_step/bin/<executable_name>']
```
OS: Debian Strech
pip list:
```# pip list
Package    Version
---------- -------
pip        19.1.1 
setuptools 41.0.1 
if submitting an ENHANCEMENT issue:
```

When running the container the executable fails with `ModuleNotFoundError: No module named 'pkg_resources'`

**Comment by ianlofs** [comment]
I also created an issue in the pipenv repository, [4128](https://github.com/pypa/pipenv/issues/4128)

**Comment by gaborbernat** [comment]
See https://github.com/pypa/virtualenv/issues/1558

**Comment by gaborbernat** [comment]
Hello, a fix for this issue has been released via virtualenv 20.0.2; see https://pypi.org/project/virtualenv/20.0.2/ (https://virtualenv.pypa.io/en/latest/changelog.html#v20-0-2-2020-02-11) . Please give a try and report back if your issue has not been addressed; if not, please comment here, and we'll reopen the ticket. We want to apologize for the inconvenience this has caused you and say thanks for having patience while we resolve the unexpected bugs with this new major release.

![thanks](https://media.tenor.co/images/1114067881329bb4218880eeb7ec34fd/tenor.gif)
